packages/paste-core/components/in-page-navigation/stories/vertical.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import type { StoryFn } from "@storybook/react";
+import { Box } from "@twilio-paste/box";
+import { CustomizationProvider } from "@twilio-paste/customization";
+import { useTheme } from "@twilio-paste/theme";
+import { useUID } from "@twilio-paste/uid-library";
+import * as React from "react";
+
+import { InPageNavigation, InPageNavigationItem } from "../src";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "Components/In Page Navigation/vertical",
+  component: InPageNavigation,
+};
+
+export const Default: StoryFn = () => {
+  /* using UID here to make unique labels for landmarks in Storybook for axe testing */
+  return (
+    <InPageNavigation aria-label={`get started ${useUID()}`} orientation="vertical">
+      <InPageNavigationItem href="#" currentPage>
+        Super SIM
+      </InPageNavigationItem>
+      <InPageNavigationItem href="#">Programmable Wireless</InPageNavigationItem>
+    </InPageNavigation>
+  );
+};
+
+export const FullWidth: StoryFn = () => {
+  /* using UID here to make unique labels for landmarks in Storybook for axe testing */
+  return (
+    <InPageNavigation aria-label={`privacy ${useUID()}`} variant="fullWidth" orientation="vertical">
+      <InPageNavigationItem href="#" currentPage>
+        Home
+      </InPageNavigationItem>
+      <InPageNavigationItem href="#">Detection</InPageNavigationItem>
+      <InPageNavigationItem href="#">Settings</InPageNavigationItem>
+    </InPageNavigation>
+  );
+};
+
+export const Inverse: StoryFn = () => {
+  /* using UID here to make unique labels for landmarks in Storybook for axe testing */
+  return (
+    <Box backgroundColor="colorBackgroundInverse" padding="space60">
+      <InPageNavigation aria-label={`privacy ${useUID()}`} variant="inverse" orientation="vertical">
+        <InPageNavigationItem href="#" currentPage>
+          Home
+        </InPageNavigationItem>
+        <InPageNavigationItem href="#">Detection</InPageNavigationItem>
+        <InPageNavigationItem href="#">Settings</InPageNavigationItem>
+      </InPageNavigation>
+    </Box>
+  );
+};
+
+export const InverseFullWidth: StoryFn = () => {
+  /* using UID here to make unique labels for landmarks in Storybook for axe testing */
+  return (
+    <Box backgroundColor="colorBackgroundInverse" padding="space60">
+      <InPageNavigation aria-label={`privacy ${useUID()}`} variant="inverse_fullWidth" orientation="vertical">
+        <InPageNavigationItem href="#" currentPage>
+          Home
+        </InPageNavigationItem>
+        <InPageNavigationItem href="#">Detection</InPageNavigationItem>
+        <InPageNavigationItem href="#">Settings</InPageNavigationItem>
+      </InPageNavigation>
+    </Box>
+  );
+};
+
+export const LinkOverflowExample: StoryFn = () => {
```

Comment:
suggestion: this story doesn't feel super useful since nothing is long enough to overflow. maybe you could wrap them in a box with a max-width to show them wrapped?

Reply:
Good callout. Fixed